app/Http/Controllers/User/BlogController.php

```diff
@@ -23,17 +25,14 @@ public function defineUser() {
      */
     public function index()
     {
-        $blogs = Blog::all();
-        if ($this->defineUser()->can_delete === Blog::IS_TRUE) {
-            $canDelete = true;
-        }else {
-            $canDelete = false;
-        }
+        $blogs = Blog::where('user_id', '=', $this->defineUser()->id)->get();
+        $canDelete = $this->defineUser()->can_delete === Blog::IS_TRUE;
         $canSee = $this->defineUser()->can_see;
-        if($canSee === Blog::IS_TRUE)
+        if ($canSee === Blog::IS_TRUE) {
             return view('blog/index', ['blogs'=>$blogs, 'canDelete'=>$canDelete]);
-        else
-            return redirect()->back();
+        } else {
+            return redirect()->back()->withErrors("you can't to enter blog manage");
```

Comment:
if this condition only use $canSee , move it to top of the function and do the return from there